src/scaffoldmaker/utils/tubemesh.py

```diff
@@ -243,28 +243,27 @@ def warpSegmentPoints(xList, d1List, d2List, segmentAxis, sx, sd1, sd2, elements
 
     return xWarpedList, d1WarpedList, d2WarpedListFinal, d3WarpedUnitList
 
-def getCoordinatesFromInner(xInner, d1Inner, d2Inner, d3Inner,
+def extrudeSurfaceCoordinates(xSurf, d1Surf, d2Surf, d3Surf,
     wallThicknessList, relativeThicknessList, elementsCountAround,
-    elementsCountAlong, elementsCountThroughWall, transitElementList):
+    elementsCountAlong, elementsCountThroughWall, transitElementList, outward=True):
     """
-    Generates coordinates from inner to outer surface using coordinates
-    and derivatives of inner surface.
-    :param xInner: Coordinates on inner surface
-    :param d1Inner: Derivatives on inner surface around tube
-    :param d2Inner: Derivatives on inner surface along tube
-    :param d3Inner: Derivatives on inner surface through wall
+    Generates extruded coordinates using coordinates and derivatives of a surface.
+    :param xSurf: Coordinates on surface
+    :param d1Surf: Derivatives on surface around tube
+    :param d2Surf: Derivatives on surface along tube
+    :param d3Surf: Derivatives on surface through wall
     :param wallThicknessList: Wall thickness for each element along tube
```

Comment:
Is this actually for each *node* along tube?

Reply:
I think this represents the wallthickness for each layer of elements along tube.